tests/src/Traits/OgTrait.php

```diff
@@ -70,4 +72,33 @@ protected function convertOgRoleNamesToIds(array $roles, EntityInterface $entity
     return $roles;
   }
 
+  /**
+   * Asserts the ownership of a user to a given group.
+   *
+   * An ownership is defined as having a specific collection of roles in that
```

Comment:
I don't like the word collection. Maybe "set of roles" or "having specific roles in that..."?